src/commands/cmd_tdigest.cc

```diff
@@ -176,6 +176,54 @@ class CommandTDigestAdd : public Commander {
   std::vector<double> values_;
 };
 
+class CommandTDigestRevRank : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    key_name_ = args[1];
+
+    std::unordered_set<std::string> unique_inputs_set(args.begin() + 2, args.end());
```

Comment:
```suggestion
    std::set<std::string> unique_inputs_set(args.begin() + 2, args.end());
```

We don't need an unordered one here, a normal `std::set` should be ok.

Reply:
@LindaSummer Yeah, std::set is ok. 

This is my first PR for a large project, and I've learned a lot. Thank you😊.